routes/patient.js

```diff
@@ -25,56 +22,22 @@ router.post('/patient', (req, res, next) => {
 		}
 
 		//Add a NextOfKin
-		let queryText = `
-			INSERT 
-			INTO HMS-PMS.NextOfKin
-			VALUES (
-			'${req.body.nextOfKinID}',
-			'${req.body.relationship}',
-			'${req.body.address}',
-			'${req.body.phoneNumber}'
-			);
-			`;
-
-		//Add a Patient
-		queryText += `
+		const query =  `
 			INSERT 
 			INTO HMS-PMS.Patient
 			VALUES (
 			'${req.body.patientID}',
```

Comment:
Missing patient's firstName, lastName, externalDoctorID

Reply:
I did not add any foreign keys for now.  will update all queries once I have the DB in front of me